src/services/enrollment.service.ts

```diff
@@ -32,15 +28,12 @@ export const getStudentEnrollments = async (
 
 export const getClassEnrollments = async (
 	subjectId: number,
-	enrollStatuses?: EnrollStatus[],
+	enrollStatuses: EnrollStatus[],
 ) => {
-	let statuses = '';
-	if (enrollStatuses) {
-		statuses += `&statuses=${enrollStatuses[0]}`;
-		for (let i = 1; i < enrollStatuses.length; i++) {
-			statuses += `%2C${enrollStatuses[i]}`;
-		}
-	}
+	const statuses = enrollStatuses
+		? `&statuses=${enrollStatuses.join('%2C')}`
+		: '';
+
```

Comment:
nadal nie wiem czemu tutaj robisz `%2C` zamiast po prostu `,`
Wydaje mi się, że next powinien ogarnąć i sam zamienić `,` na `%2C` a nawet jeśli nie to niżej przy budowaniu linku możesz dać `encodeURIComponent()`